builds/e2e/templates/lock-test-agents.yaml

```diff
@@ -0,0 +1,22 @@
+parameters:
+  upstream.protocol: 'amqp'
+  testRunnerCount: '1'
+
+jobs:
+  - job: Lock_Nested_Agents_${{ parameters['upstream.protocol'] }}
+    displayName: Lock nested agents for ${{ parameters['upstream.protocol'] }}
+    timeoutInMinutes: 180
+    pool:
+     name: $(pool.name)
+     demands:
+       - agent-group -equals $(agent.group)
+       - Agent.OS -equals Linux
+       - Agent.OSArchitecture -equals X64
+       - status -equals unlocked
+    steps:
+      - template: nested-get-secrets.yaml
+      - script: scripts/linux/nestedAgentLock.sh -a "$(agent.group)" -b "$(Build.BuildId)" -n ${{ parameters['testRunnerCount'] }} -u ${{ parameters['upstream.protocol'] }}
+        env:
+          PAT: "$(edgebuild-1-PAT)"
+        displayName: Lock agents for nested topology
+        name: lock_test_agent
```

Comment:
Is there no need for similar changes to unlock-test-agents.yaml?

Reply:
It is already a yaml. I created it in the last PR:
https://github.com/Azure/iotedge/pull/5276/files#diff-0260d575ca08d14134e4ba819a91fd93343875d12920e8c61e87f4744811b8eaR1